.changeset/long-trainers-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": minor
+---
+
+Use the `forceVerticalContainerSize` prop on `FieldContainer` to define below which container size the `vertical` styling is applied when using the `horizontal` variant
```

Comment:
```suggestion
Add a `forceVerticalContainerSize` prop to `FieldContainer` 

Use it to define below which container size the `vertical` styling is applied when using the `horizontal` variant.
```

Alternative that highlights the addition of a new prop.

Reply:
Way better 👍🏼